test/methods.js

```diff
@@ -230,7 +230,7 @@ describe('nemo-view @methods@', function () {
     util.waitForJSReady(nemo).then(function () {
       start = Date.now();
     });
-    nemo.view._waitVisible('#foo.bar.brrao', 3000).then(function (find) {
+    nemo.view._waitVisible('#foo.bar.brrao', 3000, 'Element did not load for specified timeout').then(function (find) {
```

Comment:
Oh good call. Actually I found a small bug in drivex, that needs to be merged first though - https://github.com/grawk/selenium-drivex/pull/2